src/Akka.Persistence.Sql.Hosting.Tests/CustomSqlEndToEndSpec.cs

```diff
@@ -0,0 +1,275 @@
+            var tables = await GetTableNamesAsync(_fixture.ConnectionString);
+
+            tables.Should().Contain("journal");
+            tables.Should().Contain("tags");
+            tables.Should().Contain("snapshot");
+            tables.Should().Contain("journal2");
+            tables.Should().Contain("tags2");
+            tables.Should().Contain("snapshot2");
+        }
+
+        private static async Task<string[]> GetTableNamesAsync(string connectionString)
+        {
+            await using var conn = new SQLiteConnection(connectionString);
+            await conn.OpenAsync();
+            
+            var cmd = new SQLiteCommand("SELECT name FROM sqlite_schema WHERE type = 'table' AND name NOT LIKE 'sqlite_%'", conn);
+            var reader = await cmd.ExecuteReaderAsync();
+            var tables = new List<string>();
+            while (await reader.ReadAsync())
+            {
+                tables.Add(reader.GetString(0));
+            }
+            await reader.CloseAsync();
+            
+            return tables.ToArray();
+        }
+
+        private sealed class MyPersistenceActor : ReceivePersistentActor
+        {
+            private List<int> _values = new();
+            private IActorRef? _sender;
+
+            public MyPersistenceActor(string persistenceId)
+            {
+                PersistenceId = persistenceId;
+                JournalPluginId = "akka.persistence.journal.sql";
+                SnapshotPluginId = "akka.persistence.snapshot-store.sql";
+
+                Recover<SnapshotOffer>(offer =>
+                {
+                    if (offer.Snapshot is IEnumerable<int> ints)
+                        _values = new List<int>(ints);
+                });
+
+                Recover<int>(_values.Add);
+
+                Command<int>( i =>
+                {
+                    _sender = Sender;
+                    Persist(
+                        i,
+                        _ =>
+                        {
+                            _values.Add(i);
+                            if (LastSequenceNr % 2 == 0)
+                                SaveSnapshot(_values);
+                            _sender.Tell(Ack);
+                        });
+                });
+
+                Command<string>(str => str.Equals(GetAll), _ => Sender.Tell(_values.ToArray()));
+
+                Command<SaveSnapshotSuccess>(_ => _sender.Tell(SnapshotAck));
+            }
+
+            public override string PersistenceId { get; }
+        }
+        
+        private sealed class MyCustomPersistenceActor : ReceivePersistentActor
+        {
+            private List<int> _values = new();
+            private IActorRef? _sender;
+
+            public MyCustomPersistenceActor(string persistenceId)
+            {
+                PersistenceId = persistenceId;
+                JournalPluginId = "akka.persistence.journal.custom";
+                SnapshotPluginId = "akka.persistence.snapshot-store.custom";
```

Comment:
The second persistence actor uses the non-default custom journal and snapshot plugin